test/test-deride.js

```diff
@@ -360,6 +360,74 @@ describe('Properties', function() {
     });
 });
 
+describe('Exact', function () {
+    var bob;
+    beforeEach(function (done) {
+        bob = deride.stub(['greet']);
+        done();
+    });
+
+    it('with a primative string', function (done) {
+        bob.greet('bob');
+        bob.expect.greet.called.exact('bob');
+        done();
+    });
+
+    it('with multiple a primative strings', function (done) {
+        bob.greet('alice', 'carol');
+        bob.expect.greet.called.exact('alice', 'carol');
+        done();
+    });
+
+    it('with mixed strings, arrays and numbers', function (done) {
+        bob.greet('alice', ['carol'], 123);
+        bob.expect.greet.called.exact('alice', ['carol'], 123);
+        done();
+    });
+
+    it('with mixture of primatives and objects', function () {
+        bob.greet('alice', ['carol'], 123, {
+            name: 'bob',
+            a: 1
+        }, 'sam');
+        bob.expect.greet.called.exact('alice', ['carol'], 123, {
+            name: 'bob',
+            a: 1
+        }, 'sam');
+    });
+
+    it('with a frozen object', function () {
+        bob.greet(Object.freeze({
+            name: 'bob',
+            a: 1
+        }), 'sam');
+        bob.expect.greet.called.exact(Object.freeze({
+            name: 'bob',
+            a: 1
+        }), 'sam');
+    });
+    
+    it('with a new instance', function () {
+        var Obj = function() {
+            return {
+                times: function (arg) {
+                    return arg;
+                }
+            };
+        };
+        bob.greet(new Obj());
+        bob.expect.greet.called.exact(new Obj());
```

Comment:
I suppose the issue is how I read `bob.expect.greet.called.exact()` would be I am expecting it called with the instance. It might be the difference in naming of this method: 
```
bob.expect.greet.called.exactlyMatching();
bob.expect.greet.called.matchExactly();
```
these sound less like an instance assertion